yb-voyager/cmd/importDataState.go

```diff
@@ -796,7 +806,25 @@ type Batch struct {
 }
 
 func (batch *Batch) Open() (*os.File, error) {
-	return os.Open(batch.FilePath)
+	file, err := os.Open(batch.FilePath)
```

Comment:
nit: the original `return os.Open(batch.FilePath)` suffices here, right?

Reply:
yes. redundant code.